django/bossspatialdb/views.py

```diff
@@ -79,13 +79,22 @@ def get(self, request, collection, experiment, channel, resolution, x_range, y_r
         else:
             iso = False
 
+        #Default access_mode to cache on the server side in case it is not explicitly defined in an API call. 
+        access_mode = "cache"
+        #Translation from no-cache boolean param to access_mode param for backwards compatability. 
         if "no-cache" in request.query_params:
             if request.query_params["no-cache"].lower() == "true":
-                no_cache = True
+                access_mode = "no_cache"
+            elif request.query_params["no-cache"].lower() == "false":
+                access_mode = "cache"
+                
+        if "access_mode" in request.query_params:
+            if request.query_params["access_mode"].lower() == "raw":
+                access_mode = "raw"
+            elif request.query_params["access_mode"].lower() == "no-cache":
+                access_mode = "no_cache"
             else:
-                no_cache = False
-        else:
-            no_cache = False
+                access_mode = "cache"
```

Comment:
You default to `cache` but what if `access_mode` is something other than `cache`? Should it be a `400 Bad Request` response?

Reply:
Done